src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/IActiveEditorContextTracker.cs

```diff
@@ -20,6 +20,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
     ///         tracked via <see cref="IActiveWorkspaceProjectContextHost"/>.
     ///     </para>
     /// </remarks>
+    [ProjectSystemContract(ProjectSystemContractScope.UnconfiguredProject, ProjectSystemContractProvider.Private, Cardinality = ImportCardinality.ExactlyOne)]
```

Comment:
This is not enforced "yet" so it's just documenting what scope this is supposed to live. Coincidentally I was reviving a branch early this week I had with a set of tests that would have caught this and will respect this attribute.